cfep-26.md

```diff
@@ -0,0 +1,118 @@
+
+<table>
+<tr><td> Title </td><td> Spending conda-forge project funds </td>
+<tr><td> Status </td><td> Draft  </td></tr>
+<tr><td> Author(s) </td><td> Tania Allard &lt;taniar.allard@gmail.com&gt;, Jaime Rodríguez Guerra &lt;xxxxxxxxx@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Jun 13, 2025</td></tr>
+<tr><td> Updated </td><td> Jun 13, 2025</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy for how to spend unrestricted conda-forge project funds.
+It aims to provide a set of transparent principles and guidelines for decision-making
+around the transparent use of and administration of these funds, ensuring that they are used
+in a way that benefits the conda-forge community.
+
+## Motivation
+
+conda-forge is a NumFOCUS (Austin, TX based non-profit 501(c)(3)) fiscally sponsored project,
+as such, the conda-forge project has to align with accounting and legal regulations for US-based non-profits.
+Under these considerations, non-profit donations are classified into two categories:
+**unrestricted funds** which may be used for any legal purpose appropriate to the
+organization and **restricted funds**, monies set aside for a particular purpose or
+program (for example, project-specific deliverables, educational programs, etc.).
+
+Up to now, conda-forge remains a largely volunteer-run project without significant
```

Comment:
Just to have some numbers, what amount would be significant?

Reply:
Hand-wavy: something around enough funds to hire a developer in residence. If you need more precise numbers, I can dig something up.